ui/lib/pki/addon/components/pki-not-valid-after-form.hbs

```diff
@@ -27,7 +27,14 @@
     {{/if}}
   </div>
 </div>
-<div class="column is-narrow is-flex-center has-text-grey has-right-margin-s pki-radiogroup-label">
+<div
+  class="column is-narrow is-flex-align-baseline has-text-grey has-right-margin-s
```

Comment:
What are your thoughts on removing the conditional and just adding `has-top-margin-negative-s` here?
```suggestion
  class="column is-narrow is-flex-align-baseline has-text-grey has-right-margin-s has-top-margin-negative-s
```

I'll defer to you! The the difference in the UI looked fairly negligible to me and so the conditional felt like added complexity (same for with vs without `has-padding-xs`)

But I know you've been chatting to design!

Reply:
Without the `has-top-margin-negative-s`, there is too much spacing between `TTL` and `Specific Date`. Going to keep this here since this was version I showed Ivana